packages/client/components/Transactions.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Text, HStack, Box, VStack } from "native-base";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { StyleSheet } from "react-native";
+import { Transaction } from "@cryptify/common/src/domain/entities/transaction";
+import { faCircleArrowDownLeftCustom } from "./icons/faCircleArrowDownLeftCustom";
+import { faCircleArrowUpRightCustom } from "./icons/faCircleArrowUpRightCustom";
+
+type Props = {
+    transactions: Transaction;
+    wallet: string;
+};
+
+export function FormatTransaction({ transactions, wallet }: Props) {
```

Comment:
Name the file and the component the same thing

```suggestion
export function Transaction({ transactions, wallet }: Props) {
```